packages/sdk/src/index.ts

```diff
@@ -1,21 +1,32 @@
-import _ from 'lodash';
-import BN from 'bn.js';
-import type { SubmittableExtrinsic } from '@polkadot/api/types';
-import { u8aToHex } from '@polkadot/util';
-import type { HexString } from '@polkadot/util/types';
-import type { KeyringPair } from '@polkadot/keyring/types';
-import type { ApiPromise } from '@polkadot/api';
-import { ChainAdapter, OakAdapter, TaskSchedulerChainAdapter, SendExtrinsicResult, XcmInstructionNetworkType } from '@oak-network/adapter';
-import { Weight } from '@oak-network/sdk-types';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Please DO NOT blindly add this `eslint-disable` without understanding its cause. Reference: 
https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-extraneous-dependencies.md

These libraries are not defined in the `dependencies` block of `package.json`! Please fix the [package.json](https://github.com/OAK-Foundation/oak.js/blob/dev/packages/sdk/package.json) and remove these code comments!

Reply:
Fixed